tests/store/api/test_base.py

```diff
@@ -5,11 +5,49 @@
 from dataclasses import astuple
 
 from alchy import Query, ModelBase
-
+from cg.constants.subject import PhenotypeStatus
 from cg.store.api.base import BaseHandler
 
 
 @pytest.mark.parametrize("table", astuple(BaseHandler()))
 def test__get_query(base_store, table: Type[ModelBase]):
     """Tests the _get_query function for all attributes of BaseHandler ie tables in the database."""
     assert isinstance(base_store._get_query(table=table), Query)
+
+
+def test_get_latest_analyses_for_cases_query(
+    analysis_store, helpers, timestamp_now, timestamp_yesterday
+):
+    """Tests that analyses that are not latest are not returned."""
+
+    # GIVEN an analysis that is not delivery reported but there exists a newer analysis
```

Comment:
what does `is not delivery reported` mean, and why does it matter here?

Reply:
```suggestion
    # GIVEN an analysis a newer analysis exists
```